src/case-swapping.js

```diff
@@ -15,7 +15,9 @@
  * @return {string}
  */
 function swapCase(phrase) {
-  // write code here
+  return phrase.replace(/\S/g, c => {
+    return c === c.toUpperCase() ? c.toLowerCase() : c.toUpperCase();
```

Comment:
Что такое `c`?

Reply:
Из названия переменной должно быть ясно что она собой представляет

Reply:
```suggestion
    return c === c.toUpperCase()
      ? c.toLowerCase() 
      : c.toUpperCase();
```
Переноси на новые строки